src/index.ts

```diff
@@ -236,6 +238,8 @@ class Spanner extends GrpcService {
   static GOOGLE_STANDARD_SQL =
     google.spanner.admin.database.v1.DatabaseDialect.GOOGLE_STANDARD_SQL;
 
+  emulatorHost: {endpoint: string; port?: number | undefined} | undefined;
```

Comment:
This field is probably not needed, see my comment further down.

Reply:
Removed